### DISCUSSION TITLE
Doc prefix

### DISCUSSION THREAD

**Discussion by albertodiaz** [prowide/prowide-iso20022#32] [Q&A] [answered]
Hello,

Is it possible to remove the "Doc:" prefix from the output when the getMessage() method is used? I would like to get:

`<Document xmlns="urn:iso:std:iso:20022:tech:xsd:pain.001.001.08" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">`

instead of 
`<Doc:Document ...`

**Comment by graghsaka** [comment, marked as answer]
```
MxWriteConfiguration mxWriteConfiguration = new MxWriteConfiguration();
mxWriteConfiguration.documentPrefix = null;
mxWriteConfiguration.headerPrefix = null;
System.out.println(xml.message(mxWriteConfiguration));
```
